src/include/sof/drivers/sai.h

```diff
@@ -225,7 +225,11 @@
 #define SAI_FLAG_PMQOS   BIT(0)
 
 #define SAI_FIFO_WORD_SIZE	64
-#define SAI_CLOCK_DIV		0x3
+/* Divides down the audio master clock to generate the bit clock when
+ * configured for an internal bit clock.
+ * The division value is (DIV + 1) * 2.
+ */
```

Comment:
I believe mentioning why you need to divide by 16 (if I understood this explanation correctly) might be nice, with the hardcoded params.

Reply:
This define is used by @jlacla commit fb068e6a7a1670d3f15032f27651fbe70851bccb
for now as in future will be determined dinamically

Reply:
Fair. Still, in the future better explanations are warranted. @dbaluta Is this ready to merge?